apps/cloudfront-functions/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "cloudfront-functions",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "clean": "shx rm -rf dist/",
+    "compile": "npm run clean && tsc",
+    "lint": "eslint src",
+    "test": "npm run compile && jest -i"
+  },
+  "devDependencies": {
+    "@types/aws-cloudfront-function": "^1.0.2",
+    "@types/jest": "^29.5.11",
+    "@types/rewire": "^2.5.28",
+    "eslint": "^8.40.0",
+    "eslint-config-custom": "*",
+    "jest": "^29.5.0",
+    "rewire": "^6.0.0",
+    "shx": "^0.3.4",
+    "ts-jest": "^29.1.0",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.0.2"
```

Comment:
I'd update some dependencies here: since this is a brand new app, I think we could use the latest version of most dependencies list here.

I saw these updates:
```
@types/aws-cloudfront-function   ^1.0.2  →   ^1.0.4
@types/rewire                   ^2.5.28  →  ^2.5.30
eslint                          ^8.40.0  →  ^8.56.0
jest                            ^29.5.0  →  ^29.7.0
rewire                           ^6.0.0  →   ^7.0.0
ts-jest                         ^29.1.0  →  ^29.1.1
ts-node                         ^10.9.1  →  ^10.9.2
typescript                       ^5.0.2  →   ^5.3.3
```